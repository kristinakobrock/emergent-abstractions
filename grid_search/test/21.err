  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.56it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 26.17it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 24.12it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 24.63it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 23.30it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 25.11it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.00it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 24.71it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 23.87it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 23.25it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 23.62it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 26.59it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 25.17it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 25.24it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 25.25it/s]100%|############################################################################################| 50/50 [00:01<00:00, 28.85it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.38it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.95it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 27.83it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 26.98it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.90it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.92it/s]
