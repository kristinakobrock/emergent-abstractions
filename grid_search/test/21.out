# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6751733422279358, "acc": 0.6152791976928711, "length": 3.5590591430664062, "mode": "train", "epoch": 1}
{"loss": 0.6699509620666504, "acc": 0.624804675579071, "length": 2.4765625, "mode": "test", "epoch": 1}
{"loss": 0.6651994585990906, "acc": 0.6261093020439148, "length": 3.489710569381714, "mode": "train", "epoch": 2}
{"loss": 0.672553539276123, "acc": 0.6140625476837158, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6624255180358887, "acc": 0.6252319812774658, "length": 3.3951497077941895, "mode": "train", "epoch": 3}
{"loss": 0.6676938533782959, "acc": 0.6248047351837158, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6600728631019592, "acc": 0.6271873116493225, "length": 3.2500269412994385, "mode": "train", "epoch": 4}
{"loss": 0.6633884310722351, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6587278246879578, "acc": 0.6289377808570862, "length": 3.1489429473876953, "mode": "train", "epoch": 5}
{"loss": 0.6654646396636963, "acc": 0.6238220930099487, "length": 1.10546875, "mode": "test", "epoch": 5}
{"loss": 0.6560877561569214, "acc": 0.6282141804695129, "length": 3.05629301071167, "mode": "train", "epoch": 6}
{"loss": 0.6521778702735901, "acc": 0.628227174282074, "length": 2.2421875, "mode": "test", "epoch": 6}
{"loss": 0.6500139236450195, "acc": 0.6313779354095459, "length": 3.0975430011749268, "mode": "train", "epoch": 7}
{"loss": 0.6438195109367371, "acc": 0.6329237818717957, "length": 2.76953125, "mode": "test", "epoch": 7}
{"loss": 0.6375539898872375, "acc": 0.6408544182777405, "length": 3.2402091026306152, "mode": "train", "epoch": 8}
{"loss": 0.627456545829773, "acc": 0.6589202880859375, "length": 3.26171875, "mode": "test", "epoch": 8}
{"loss": 0.6190424561500549, "acc": 0.6629397869110107, "length": 3.389871120452881, "mode": "train", "epoch": 9}
{"loss": 0.6244884729385376, "acc": 0.6572234630584717, "length": 3.80078125, "mode": "test", "epoch": 9}
{"loss": 0.6038540005683899, "acc": 0.6781163215637207, "length": 3.5920908451080322, "mode": "train", "epoch": 10}
{"loss": 0.6293501853942871, "acc": 0.6504501104354858, "length": 3.7890625, "mode": "test", "epoch": 10}
{"loss": 0.5928267240524292, "acc": 0.6847134232521057, "length": 3.6418850421905518, "mode": "train", "epoch": 11}
{"loss": 0.6124448180198669, "acc": 0.6643692255020142, "length": 3.56640625, "mode": "test", "epoch": 11}
{"loss": 0.5809498429298401, "acc": 0.6971520781517029, "length": 3.5368528366088867, "mode": "train", "epoch": 12}
{"loss": 0.6089925765991211, "acc": 0.6774002313613892, "length": 3.58203125, "mode": "test", "epoch": 12}
{"loss": 0.5645104050636292, "acc": 0.7136000394821167, "length": 3.611137628555298, "mode": "train", "epoch": 13}
{"loss": 0.5980916023254395, "acc": 0.6887816786766052, "length": 3.66015625, "mode": "test", "epoch": 13}
{"loss": 0.5478572249412537, "acc": 0.7241926193237305, "length": 3.6480655670166016, "mode": "train", "epoch": 14}
{"loss": 0.595159113407135, "acc": 0.7015746831893921, "length": 3.61328125, "mode": "test", "epoch": 14}
{"loss": 0.5317426323890686, "acc": 0.7330080270767212, "length": 3.639108657836914, "mode": "train", "epoch": 15}
{"loss": 0.5976873636245728, "acc": 0.7076811790466309, "length": 3.67578125, "mode": "test", "epoch": 15}
{"loss": 0.5141978859901428, "acc": 0.7502818703651428, "length": 3.682192325592041, "mode": "train", "epoch": 16}
{"loss": 0.5844342112541199, "acc": 0.7183974981307983, "length": 3.7890625, "mode": "test", "epoch": 16}
{"loss": 0.4980064928531647, "acc": 0.7603121995925903, "length": 3.7351036071777344, "mode": "train", "epoch": 17}
{"loss": 0.5911153554916382, "acc": 0.7291977405548096, "length": 3.8671875, "mode": "test", "epoch": 17}
{"loss": 0.48141494393348694, "acc": 0.7689522504806519, "length": 3.800285577774048, "mode": "train", "epoch": 18}
{"loss": 0.591571569442749, "acc": 0.732226550579071, "length": 3.9609375, "mode": "test", "epoch": 18}
{"loss": 0.46995872259140015, "acc": 0.7728875875473022, "length": 3.8931996822357178, "mode": "train", "epoch": 19}
{"loss": 0.5936051607131958, "acc": 0.729095458984375, "length": 3.984375, "mode": "test", "epoch": 19}
{"loss": 0.45926642417907715, "acc": 0.7793518900871277, "length": 3.933405637741089, "mode": "train", "epoch": 20}
{"loss": 0.5865108966827393, "acc": 0.7406158447265625, "length": 3.98828125, "mode": "test", "epoch": 20}
{"loss": 0.45144015550613403, "acc": 0.7838614583015442, "length": 3.934130907058716, "mode": "train", "epoch": 21}
{"loss": 0.576850175857544, "acc": 0.7457306385040283, "length": 3.9921875, "mode": "test", "epoch": 21}
{"loss": 0.4474930167198181, "acc": 0.7886933088302612, "length": 3.937998056411743, "mode": "train", "epoch": 22}
{"loss": 0.564251720905304, "acc": 0.7509949207305908, "length": 3.9921875, "mode": "test", "epoch": 22}
{"loss": 0.43840476870536804, "acc": 0.7949223518371582, "length": 3.9490151405334473, "mode": "train", "epoch": 23}
{"loss": 0.5767168998718262, "acc": 0.7527313828468323, "length": 3.99609375, "mode": "test", "epoch": 23}
{"loss": 0.43574953079223633, "acc": 0.7927576899528503, "length": 3.953012466430664, "mode": "train", "epoch": 24}
{"loss": 0.567631721496582, "acc": 0.747843861579895, "length": 3.9921875, "mode": "test", "epoch": 24}
{"loss": 0.43250176310539246, "acc": 0.7973004579544067, "length": 3.9537277221679688, "mode": "train", "epoch": 25}
{"loss": 0.5772552490234375, "acc": 0.7476745843887329, "length": 3.9921875, "mode": "test", "epoch": 25}
{"loss": 0.42411547899246216, "acc": 0.8026521801948547, "length": 3.952887773513794, "mode": "train", "epoch": 26}
{"loss": 0.5620458126068115, "acc": 0.7509765625, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.42336755990982056, "acc": 0.8071501851081848, "length": 3.954094648361206, "mode": "train", "epoch": 27}
{"loss": 0.5753487944602966, "acc": 0.7513671517372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4180373549461365, "acc": 0.8073685169219971, "length": 3.959667921066284, "mode": "train", "epoch": 28}
{"loss": 0.578622043132782, "acc": 0.7533202767372131, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4132250249385834, "acc": 0.8120319247245789, "length": 3.9625914096832275, "mode": "train", "epoch": 29}
{"loss": 0.5642789602279663, "acc": 0.7511719465255737, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4073403477668762, "acc": 0.8137695789337158, "length": 3.9652345180511475, "mode": "train", "epoch": 30}
{"loss": 0.5532925128936768, "acc": 0.7683592438697815, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.404617577791214, "acc": 0.8142626285552979, "length": 3.966357707977295, "mode": "train", "epoch": 31}
{"loss": 0.5485630035400391, "acc": 0.7658203840255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3993537724018097, "acc": 0.8185368776321411, "length": 3.972283124923706, "mode": "train", "epoch": 32}
{"loss": 0.5561208724975586, "acc": 0.7625000476837158, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3948739171028137, "acc": 0.8242191076278687, "length": 3.978410243988037, "mode": "train", "epoch": 33}
{"loss": 0.547808825969696, "acc": 0.7679687142372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.38871821761131287, "acc": 0.8286160826683044, "length": 3.975963830947876, "mode": "train", "epoch": 34}
{"loss": 0.5504417419433594, "acc": 0.7650390267372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.38236430287361145, "acc": 0.8292778730392456, "length": 3.977477788925171, "mode": "train", "epoch": 35}
{"loss": 0.5588406920433044, "acc": 0.7685546875, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3761923611164093, "acc": 0.8333298563957214, "length": 3.9797909259796143, "mode": "train", "epoch": 36}
{"loss": 0.5484281182289124, "acc": 0.7705078125, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.37082409858703613, "acc": 0.836337685585022, "length": 3.9854393005371094, "mode": "train", "epoch": 37}
{"loss": 0.5333480834960938, "acc": 0.7730469107627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3692096173763275, "acc": 0.8373538255691528, "length": 3.9872682094573975, "mode": "train", "epoch": 38}
{"loss": 0.5264632701873779, "acc": 0.7732421159744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3660106360912323, "acc": 0.8383461236953735, "length": 3.9904541969299316, "mode": "train", "epoch": 39}
{"loss": 0.5205529928207397, "acc": 0.7738280892372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3614697754383087, "acc": 0.8398666381835938, "length": 3.9878580570220947, "mode": "train", "epoch": 40}
{"loss": 0.5178539156913757, "acc": 0.7857422232627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.35759568214416504, "acc": 0.8426096439361572, "length": 3.9898622035980225, "mode": "train", "epoch": 41}
{"loss": 0.5026288628578186, "acc": 0.7861327528953552, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.35149839520454407, "acc": 0.8453002572059631, "length": 3.9908738136291504, "mode": "train", "epoch": 42}
{"loss": 0.5179234743118286, "acc": 0.7876953482627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3488166630268097, "acc": 0.8473705053329468, "length": 3.993013620376587, "mode": "train", "epoch": 43}
{"loss": 0.4876026511192322, "acc": 0.7939453125, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.34325286746025085, "acc": 0.8510469198226929, "length": 3.9920501708984375, "mode": "train", "epoch": 44}
{"loss": 0.48484015464782715, "acc": 0.7941406965255737, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.34175917506217957, "acc": 0.8516596555709839, "length": 3.992863655090332, "mode": "train", "epoch": 45}
{"loss": 0.48737043142318726, "acc": 0.7978515028953552, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.33847612142562866, "acc": 0.8535122871398926, "length": 3.9904000759124756, "mode": "train", "epoch": 46}
{"loss": 0.47833871841430664, "acc": 0.7927734851837158, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.33118170499801636, "acc": 0.8557271361351013, "length": 3.992490291595459, "mode": "train", "epoch": 47}
{"loss": 0.4600214958190918, "acc": 0.8039063215255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3267712891101837, "acc": 0.8599554896354675, "length": 3.997000217437744, "mode": "train", "epoch": 48}
{"loss": 0.4614037871360779, "acc": 0.8066405653953552, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.32141655683517456, "acc": 0.8611756563186646, "length": 3.9962692260742188, "mode": "train", "epoch": 49}
{"loss": 0.47360843420028687, "acc": 0.8013672828674316, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3248719274997711, "acc": 0.8573212623596191, "length": 3.9967856407165527, "mode": "train", "epoch": 50}
{"loss": 0.4959397614002228, "acc": 0.7994140386581421, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.313239723443985, "acc": 0.8653804659843445, "length": 3.9954710006713867, "mode": "train", "epoch": 51}
{"loss": 0.4712122082710266, "acc": 0.8093748688697815, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.309611052274704, "acc": 0.8689489364624023, "length": 3.9963858127593994, "mode": "train", "epoch": 52}
{"loss": 0.45352768898010254, "acc": 0.8150390982627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3050023019313812, "acc": 0.8698548674583435, "length": 3.997577667236328, "mode": "train", "epoch": 53}
{"loss": 0.46422603726387024, "acc": 0.8128905296325684, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2997639775276184, "acc": 0.8739763498306274, "length": 3.9957666397094727, "mode": "train", "epoch": 54}
{"loss": 0.47427523136138916, "acc": 0.8189453482627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.29896387457847595, "acc": 0.8746353387832642, "length": 3.99676775932312, "mode": "train", "epoch": 55}
{"loss": 0.4645489454269409, "acc": 0.8160156011581421, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2962707579135895, "acc": 0.8759772777557373, "length": 3.9965851306915283, "mode": "train", "epoch": 56}
{"loss": 0.44131776690483093, "acc": 0.8285155296325684, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.292215496301651, "acc": 0.8785954117774963, "length": 3.9962692260742188, "mode": "train", "epoch": 57}
{"loss": 0.46180129051208496, "acc": 0.8201171159744263, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2905409336090088, "acc": 0.8785552382469177, "length": 3.99711537361145, "mode": "train", "epoch": 58}
{"loss": 0.4307563304901123, "acc": 0.8236327767372131, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.28375568985939026, "acc": 0.8825180530548096, "length": 3.9970016479492188, "mode": "train", "epoch": 59}
{"loss": 0.4373301863670349, "acc": 0.8273438215255737, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2809906005859375, "acc": 0.885753333568573, "length": 3.997288703918457, "mode": "train", "epoch": 60}
{"loss": 0.43172937631607056, "acc": 0.8289061188697815, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7161667943000793
